MIP41/mip41.md

```diff
@@ -142,29 +146,51 @@ MIP41c5 subproposals must use the template located at [MIP41c5-Subproposal-Templ
+If a new permanent Facilitator is onboarded to the relevant Core Unit through a MIP41c4 subproposal the Interim Facilitator is immediately offboarded and all powers and responsibilities are transferred to the new permanent Facilitator.
 
-In the event of a Core Unit offboarding, if the Retiring Facilitator is uncooperative, the Interim Facilitator will be in charge of overseeing/executing the unwinding process. In the event of a Facilitator offboarding, they will be in charge of ensuring the functional continuity of the Core Unit until a new permanent Facilitator is ratified.
+In the event of a Core Unit offboarding, the Interim Facilitator is responsible for overseeing and executing the unwinding process as defined in MIP39c3. 
 
-The incentive to volunteer as an Interim Facilitator is the compensation tied to a Core Unit unwinding plan. In the event of a missing, failed succession, or headless core unit, the incentive to volunteer as an Interim Facilitator is a transfer of the former Facilitator's compensation for an interim timeline.
+In the event of a Facilitator offboarding, the Interim Facilitator is responsible for both:
+* Ensuring the functional continuity of the Core Unit during the interim period.
+* Making a good faith effort to deliver one or more viable Facilitator Onboarding subproposals for the Core Unit to Maker Governance. The Interim Facilitator may propose themselves as a permanent Facilitator.
 
-If a Core Unit has no active Facilitators as a result of involuntary offboarding and has no active budget, no Interim Facilitator will be appointed unless there are winding down activities that require one.
+The Interim Facilitator is empowered to use the Facilitator Governance Powers defined in MIP41c2 to keep existing operational processes running smoothly during the interim period.
+
+If the Core Unit is not being off-boarded, an Interim Facilitator receives DAI compensation monthly equal to the salary of the previous permanent Facilitator if this information is publicly available, otherwise 15,000 DAI per month. 
+
+When a Core Unit is offboarded if an Interim Facilitator is judged to be necessary to wind down the Core Unit gracefully, then compensation for an Interim Facilitator must be included as part of the offboarding subproposal.
+
+---
+
+### MIP41c7: Interim Facilitator Appointment Process 
 
-#### The Fallback Decision Process
+The Interim Facilitator Appointment Process assumes the existence of one or more permanent Governance Facilitators. If this is not the case, MIP41c6 must be considered non-binding. The Governance Facilitators are responsible for the implementation of this process. A majority of Governance Facilitators may modify this process to suit the circumstances by communicating details and reasoning publicly before the process begins.
```

Comment:
This should be MIP41c7 now that we've split out the component, right?

```suggestion
The Interim Facilitator Appointment Process assumes the existence of one or more permanent Governance Facilitators. If this is not the case, MIP41c7 must be considered non-binding. The Governance Facilitators are responsible for the implementation of this process. A majority of Governance Facilitators may modify this process to suit the circumstances by communicating details and reasoning publicly before the process begins.
```

Reply:
Fixed, thank you.